src/createReducer.js

```diff
@@ -361,6 +361,11 @@ function createReducer<M, L>(structure: Structure<M, L>) {
         result = setIn(result, 'values', values)
         result = setIn(result, 'initial', values)
       }
+      var submitFailed = getIn(state, 'submitFailed');
```

Comment:
Hi all, 
I noticed that when form is rendered and there is form validation errors so 'SyncErrors' field is present. In case if i understand correctly - form is invalid from start.